packages/plugin-truffle/test/contracts/Constructor.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.5.0;
+
+contract WithConstructor {
```

Comment:
Please use the same contract that we use to test Hardhat. Eventually we should move to a shared set of contracts and using different contracts will make that a little more difficult.

Reply:
I'd love to ... but truffle doesn't support multiple version of soldidty used in the same repo

... the other contract are using 0.5 ...
... with 0.5 we cannot put natspec or immutable ...

... truffle is really not something I enjoy working with